lte/gateway/fabfile.py

```diff
@@ -135,6 +135,8 @@ def package(
 
         with cd('release'):
             mirrored_packages_file = 'mirrored_packages'
+            if os == "ubuntu":
```

Comment:
do we support any other os type?

Reply:
the mirrored_packages file is for stretch